File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        String s = "";
        while(columnNumber != 0){
            int y = columnNumber%26;
            columnNumber = columnNumber/26;
            if (y == 0) {
                s = String.valueOf((char) ('Z')) + s;
                if (columnNumber == 1)
                    break;
                columnNumber--;
            }
            else
                s = String.valueOf((char)(y-1+'A')) + s;
        }

        return s;
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res;       
        
        while(columnNumber){
            columnNumber -= 1;
            char temp='A' + columnNumber%26;
            res = temp + res;
            columnNumber /= 26;
        }
       
    return res;    
    }
};